doc/source/whatsnew/v0.18.1.txt

```diff
@@ -131,6 +131,7 @@ API changes
 
 
 
+- the default for ``.query()/.eval()`` is now ``engine=None`` which will use ``numexpr`` if its installed, else will fallback to the ``python`` engine. This mimics the pre-0.18.1 behavior if ``numexpr`` is installed (and which previously would raise if ``numexpr`` was NOT installed and ``.query()/.eval()`` was used). (:issue:`12749`)
```

Comment:
no reason to show a ``PerformanceWarning``. I guess we could, but the issue addressed here is thatt was raising an ``ImportError`` if you didn't have numexpr installed and DIDN't specify ```engine='python'`` which was not the default